controller/rebuild.go

```diff
@@ -76,17 +91,29 @@ func (c *Controller) VerifyRebuildReplica(address string) error {
 		return err
 	}
 
-	logrus.Infof("chain %v from rw replica %s", rwChain, rwReplica.Address)
-	// Don't need to compare the volume head disk
-	rwChain = rwChain[1:]
-
 	chain, err := getReplicaChain(address)
 	if err != nil {
 		return err
 	}
+	WOcheckpoint, err := getReplicaCheckpoint(address)
```

Comment:
can you add a comment on why WOcheckpoint should be considered but not that of RWcheckpoint?

Reply:
consider possible cases in the comment like WO checkpoint is latest than RW checkpoint and vice versa